### DISCUSSION TITLE
Google Play release

### DISCUSSION THREAD

**Discussion by Aman9das** [Arcticons-Team/Arcticons#335] [Ideas]
I think it'd be great for 
- you to have a source of revenue.

And given how polished the icon pack is, 
- draw people to open source software.

I got started by termux-styling myself, basically thought I'm fooling the devs by pirating the paid app 😆

**Comment by Donnnno** [comment]
I was actually thinking about doing this earlier, but still gotta set up a google dev account

**Reply by Aman9das** [reply]
https://commonsclause.com/

Until you publish, the app you may want to look into this..
The clause withdraws the right to "Sell" the app without adding value. Trebleshot has a bunch of clones on playstore basically adding the "advertisement" functionality.. which is detrimental for the project's visibility

**Reply by Donnnno** [reply]
This actually sounds like a really nice license, it has always been a concern that people do this stuff without adding anything to it. 
So, I may look into changing the license

**Reply by Aman9das** [reply]
Pretty nifty as it isnt violating the gpl..

**Reply by Aman9das** [reply]
But I'm no lawyer, you should talk to the fdroid people or something

**Reply by mubashir-rehman** [reply]
just in case https://github.com/fossas/commons-clause

**Reply by mubashir-rehman** [reply]
not a good option as of now to use commonclasue as per https://github.com/fossas/commons-clause/issues/11

**Reply by Donnnno** [reply]
I think it isn't compatible with F-Droid because you can't call it technically open-source according to themselves.

**Comment by Donnnno** [comment]
Working on getting this done btw, I've submitted it to Google Play, and now I have to wait if it gets approved.

**Reply by Donnnno** [reply]
Oh, it's approved after 5 days waiting. It's 2,50 euros on GPlay.
Probably putting the money into funds for other open-source projects, if people are paying for it :^) 

https://play.google.com/store/apps/details?id=com.donnnno.arcticons
https://play.google.com/store/apps/details?id=com.donnnno.arcticons.light

**Reply by Aman9das** [reply]
I think you should ask in android themes subreddit about pricing. They'll tell it correctly I think.

rn we're mostly lineX competitors, so making it pricier is bad i feel, but that's just my opinion..

**Reply by Donnnno** [reply]
Yea, maybe. But I'm not doing this to be competitive with others. I just want an open-source icon pack with a slightly bigger reach.
I personally think that 2,50 isn't much, especially when you count in all the work that has been done for it. 

But fair, I'll think about it :^)

**Reply by Donnnno** [reply]
I mean it still is free, and always will be, just not on Google Play

**Reply by Aman9das** [reply]
> 2,50 isn't much, especially when you count in all the work that has been done for it.

I know lol.. but it's about optimizing total revenue. Not making every user pay the intellectual debt

I'll look into creating variants with different colours then you can publish a fleet on play store, pretty great with android 12 around the corner..

That'd be dope :)

**Reply by Aman9das** [reply]
https://www.reddit.com/r/androidthemes/comments/lnk1gc/promotion_trippy_icons_is_live_includes_3000_icons/

The guy worked harder than us on each icon, made more icons, made more icons, is cheaper and still has less than 500 installs :(

**Reply by Donnnno** [reply]
I've lowered the price to 1.50 euros, but not gonna get any lower than that. 
Pricing is something that everyone has to decide for itself. 
I don't really care about big numbers on Google Play, it's already an over saturated marked there. F-Droid is nice in that sense, I mean, we have the biggest open-source icon pack on there. 

Honestly, I just want to earn my $25 I invested in becoming a Google Dev back :^)

**Comment by Aman9das** [comment]
The light version hasn't been updated in Play Store

**Reply by Donnnno** [reply]
Yea, gotta do that too. Had some issues with the build number. So delayed it. Feels kinda pointless to update for 0 people
